src/etcd_client.rs

```diff
@@ -83,12 +68,7 @@ impl EtcdClient {
     ) -> anyhow::Result<Vec<T>> {
         let mut req = etcd_rs::DeleteRequest::new(etcd_rs::KeyRange::key(key));
         req.set_prev_kv(true);
-        let mut resp = self.etcd_rs_client.kv().delete(req).await.map_err(|e| {
-            anyhow!(
-                "failed to get DeleteResponse from etcd, the error is: {}",
-                e,
-            )
-        })?;
+        let mut resp = self.etcd_rs_client.kv().delete(req).await?;
```

Comment:
add `.context(format!("failed to get DeleteResponse from etcd for DeleteRequest={:?}", req))?;`